bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -558,11 +565,40 @@ def make_payment_within_bank(self):
+                        "xpath",
+                    )
+                    select_imps.click()
+
+                case "Transfer to other bank (RTGS)":
+                    select_rtgs = self.get_element(
+                        "//div[contains(@class,'transfer-mode-')][.//label[normalize-space()='RTGS']]",
+                        "xpath",
+                    )
+                    select_rtgs.click()
+
+        except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)